README.md

```diff
@@ -1,239 +1,99 @@
-
-## Securing Zowe CLI profiles
+Tip: After the installation process completes, it validates that the plug-in was installed correctly and the names of its commands, options, and arguments do not conflict with that of the other plug-ins that you installed into your Zowe CLI implementation.
 
-When you install the plug-in using the Windows Installation Wizard, the plug-in secures all of your existing (Zowe CLI) profiles. Conversely, when you do not use the wizard to install the plug-in (you installed the plug-in from an online registry or from a local package), you secure your profiles manually using the Set command. For more information, see [How the plug-in works](#how-the-plug-in-works).
+When the validation process is successful, the following message displays:
 
-Issue the following `Set` command to **secure** all of your Zowe CLI profiles:
-```
-zowe config set CredentialManager @zowe/secure-credential-store-for-zowe-cli
+```bash
+Validation results for plugin '@zowe/secrets-for-kubernetes-for-zowe-cli'
+This plugin was successfully validated. Enjoy the plugin.
 ```
 
-When you want to **unsecure** all of your Zowe CLI profiles, issue the following `Reset` command:
-```
-zowe config reset CredentialManager
-```
-
-Your profiles will need to be recreated after the CredentialManager is reset.
-
-**Note:** For information about how you can customize the behavior of the plug-in, see [Overriding the Default Credential Manager with a Plugin](https://github.com/zowe/imperative/wiki/Overriding-the-Default-Credential-Manager-with-a-Plugin) in the [Imperative CLI Framework](https://github.com/zowe/imperative/wiki) documentation.
-
-## Running tests
-
-You can perform the following types of tests on the Secure Credential Store Plug-in:
-- Unit
-- Integration
-- System
-
-**Note:** For detailed information about conventions and best practices for running tests against Zowe CLI plug-ins, see see [Zowe CLI Plug-in Testing Guidelines](https://github.com/zowe/zowe-cli/blob/master/docs/PluginTESTINGGuidelines.md).
-
-Before running the system and integration tests, you must have a server connection to run against. For more information, see [Software requirements](#software-requirements).
-
-To define access credentials to the server, copy the file named `.../__tests__/__resources__/properties/example_properties.yaml` and create a file named `.../__tests__/__resources__/properties/custom_properties.yaml`.
-
-**Note:** Information about how to customize the `custom_properties.yaml` file is provided in the yaml file itself.
+When an unsuccessful message displays, you can troubleshoot the installation by addressing the issues that the message describes. You can also review the information that is contained in the log file that is located in the Zowe CLI home directory.
```

Comment:
- is it an "unsuccessful" message or is it a "failure" message?
- I would reword the sentence: " ... by addressing the issues outlined/described in the message."
- Also would tweak the next sentence: "... contained in the log file located in the Zowe CLI ..."

Reply:
it is a failure message that is displayed when something went wrong when installing the plug-in :)